clubs/models.py

```diff
@@ -71,6 +61,104 @@ class Club(models.Model):
         null=True
     )
 
+    occupation = models.TextField(
```

Comment:
Do we want to restrict the length of this field here? i.e. use a `CharField` instead?